src/functions/database/globalUserLeaderBoard.js

```diff
@@ -1,54 +1,59 @@
 module.exports = async (d) => {
     const data = d.util.aoiFunc(d);
     if (data.err) return d.error(data.err);
-    const [variable, type = "asc", custom = `{top}. {username}: {value}`, list = 10, page = 1, table = d.client.db.tables[0], hideNegativeValue = false, hideZeroValue = false ] = data.inside.splits;
+    const [variable, type = "asc", custom = `{top}. {username}: {value}`, list = 10, page = 1, table = d.client.db.tables[0], hideNegativeValue = false, hideZeroValue = false] = data.inside.splits;
 
-    if (!d.client.variableManager.has(variable, table)) return d.aoiError.fnError(d, 'custom', {}, `Variable "${variable}" Not Found`);
+    if (!d.client.variableManager.has(variable, table)) return d.aoiError.fnError(d, "custom", {}, `Variable "${variable}" Not Found`);
 
-    if (!type || (type.toLowerCase() !== "asc" && type.toLowerCase() !== "desc")) return d.aoiError.fnError(d, 'custom', {}, `type must be "desc" or "asc"`)
+    if (!type || (type.toLowerCase() !== "asc" && type.toLowerCase() !== "desc")) return d.aoiError.fnError(d, "custom", {}, `Invalid type must be "desc" or "asc"`);
 
     const result = [];
 
-    let db = await d.client.db.all(table, (data) => { return data.key.startsWith(variable.addBrackets() + "_") && data.key.split("_").length === 2 }, list * page, type);
+    let db = await d.client.db.all(
+        table,
+        (data) => {
+            return data.key.startsWith(variable.addBrackets() + "_") && data.key.split("_").length === 2;
+        },
+        list * page,
+        type
+    );
 
     let i = 1;
     for (const lbdata of db) {
-        const key = lbdata.key.split("_")[1]
+        const key = lbdata.key.split("_")[1];
         const user = await d.util.getUser(d, key);
 
-        if (!user) return d.aoiError.fnError(d, "custom", {}, `user: ${key}`);
-
         if (hideNegativeValue === "true" && parseInt(lbdata.value) < 0) continue;
         if (hideZeroValue === "true" && parseInt(lbdata.value) == 0) continue;
-  
+
         const replacer = {
             "{value}": parseInt(lbdata.value),
             "{top}": i,
-            "{username}": user.username,
-            "{displayName}": user.displayName,
-            "{tag}": user.tag,
-            "{id}": user.id,
-            "{mention}": user,
-        }
+            "{username}": user?.username || "Unknown User",
+            "{displayName}": user?.displayName || "Unknown User",
+            "{tag}": user?.tag || "Unknown User",
+            "{id}": user?.id || "Unknown User",
+            "{mention}": user || "Unknown User"
+        };
```

Comment:
just ignore all non existing user one
as uh other var types can also be mixed in 

if the user is using same var for different var type